recipes/telluric/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "telluric" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "88c92f26b75d3600d4546a533292a13ceb38a37e93e627a232b72308075fd0a8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: True
```

Comment:
With this `skip: True`, it will skip all builds. It looks like you need python>3.4. See the discussion under the bullet points in the docs:

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

Reply:
Oops, this is a leftover from the first commit.

However, I don't understand why this made the linter fail: https://github.com/conda-forge/staged-recipes/pull/5748/commits/c990c1c1e9877a798b2c72c4950ac39dabf11efd#diff-d575fe2f92afe10f1af4decbee30b445R18